pkg/indexer/types.go

```diff
@@ -30,6 +30,16 @@ type status struct {
 	LedgerIndex uint32
 }
 
+type protocol struct {
+	ID uint `gorm:"primaryKey;notnull"`
+	// The version of the protocol running
+	Version byte
```

Comment:
We don't need an entry for the version.
It should instead check if the Version we receive is supported by this plugin.